packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -449,9 +443,18 @@ export default abstract class SqlIntegration
         variation: row.variation ?? "",
         dimension: row.dimension || "",
         users: parseInt(row.users) || 0,
-        count: parseFloat(row.count) || 0,
-        mean: parseFloat(row.mean) || 0,
-        stddev: parseFloat(row.stddev) || 0,
+        count: parseInt(row.count) || 0,
+        statistic_type: row.statistic_type ?? "",
+        main_metric_type: row.main_metric_type ?? "",
+        main_sum: parseFloat(row.main_sum) || 0,
+        main_sum_squares: parseFloat(row.main_sum_squares) || 0,
+        ...(row.denominator_metric_type && {
```

Comment:
@jdorn this is where I added the filter to remove unnecessary fields from returning in the UI if the metric is only a mean metric and not a ratio metric. See this loom for the updated, desired behavior: https://www.loom.com/share/da580ba3b0c543878f5e155c70e1183a